deploy/kubernetes/base/controller/setup_cluster.yaml

```diff
@@ -17,6 +17,25 @@ metadata:
   name: gcp-filestore-csi-controller-sa
   namespace: gcp-filestore-csi-driver
 ---
+
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: csi-gcp-fs-controller
+value: 900000000
+globalDefault: false
+description: "This priority class should be used for the GCE FS CSI driver controller deployment only."
```

Comment:
I don't think there's an official abbreviation for Filestore. Maybe just spell it out to be more clear?

Reply:
Ack